ggml/src/ggml.c

```diff
@@ -225,7 +225,7 @@ static void ggml_print_backtrace(void) {
             if (WEXITSTATUS(wstatus) == EXIT_FAILURE) {
                 // gdb failed, fallback to backtrace_symbols
                 ggml_print_backtrace_symbols();
-            }
+    }
```

Comment:
erronous removal of spaces?

Reply:
Artifacts from the merge, should all have been cleaned up in 3a3a752 as mentioned by @compilade